easybuild/easyblocks/c/cuda.py

```diff
@@ -266,6 +281,11 @@ def sanity_check_step(self):
             custom_paths['files'].append(os.path.join("extras", "CUPTI", "lib64", "libcupti.%s") % shlib_ext)
             custom_paths['dirs'].append(os.path.join("extras", "CUPTI", "include"))
 
+        # Just a subset of files are checked, since the whole list is likely to change, and irrelevant in most cases
+        # anyway
+        pc_files = ['cublas.pc', 'cudart.pc', 'cuda.pc', 'nvidia-ml.pc', 'nvjpeg.pc']
+        custom_paths['files'] = custom_paths['files'] + [os.path.join('pkgconfig', x) for x in pc_files]
```

Comment:
Please use append like above instead

Reply:
Done in https://github.com/easybuilders/easybuild-easyblocks/pull/2656/commits/bcd34ce15ce09f7fb540d2ffc82f82effeee68cc

Reply:
`append` does not do that. I guess you mean `extend` (as in Kenneth's fix)